sklearn/linear_model/tests/test_coordinate_descent.py

```diff
@@ -247,6 +251,125 @@ def test_lasso_cv_positive_constraint():
     assert min(clf_constrained.coef_) >= 0
 
 
+@pytest.mark.parametrize(
+    "test_model, args",
+    [(Lasso, {"tol": 1e-16, "alpha": 0.1}), (LassoLars, {"alpha": 0.1}),
+     (RidgeClassifier, {"solver": 'sparse_cg', "alpha": 0.1}),
+     (ElasticNet, {"tol": 1e-16, 'l1_ratio': 1, "alpha": 0.1}),
+     (ElasticNet, {"tol": 1e-16, 'l1_ratio': 0, "alpha": 0.1}),
+     (Ridge, {"solver": 'sparse_cg', 'tol': 1e-12, "alpha": 0.1}),
+     (BayesianRidge, {}), (ARDRegression, {}), (OrthogonalMatchingPursuit, {}),
+     (MultiTaskElasticNet, {"tol": 1e-16, 'l1_ratio': 1, "alpha": 0.1}),
+     (MultiTaskElasticNet, {"tol": 1e-16, 'l1_ratio': 0, "alpha": 0.1}),
+     (MultiTaskLasso, {"tol": 1e-16, "alpha": 0.1}),
+     (Lars, {}), (LinearRegression, {}), (LassoLarsIC, {})
+     ])
+def test_model_pipeline_same_as_normalize_true(test_model, args):
+    # Test that linear models (test_model) set with normalize set to True are
+    # doing the same as the same linear model preceeded by StandardScaler
+    # in the pipeline and with normalize set to False
+
```

Comment:
We can start by creating the model

```python
model_normalize = LinearModel(normalize=True, fit_intercept=True, **params)
pipeline_standardize = make_pipeline(
    StandardScaler(), LinearModel(normalize=False, fit_intercept=True, **params
)
```

Reply:
arff ok. So in this case we need to delay it. In this case we need to use `issubclass` instead of `isinstance`:

```python
if issubclass(LinearModel, ClassifierMixin):
    # this will be a classifier
```

Reply:
or wait we could just a `set_params` at this moment.